internal/plugins/plugins.go

```diff
@@ -255,6 +255,9 @@ func (p Plugin) Update(conf config.Config, ref string, out, errout io.Writer) (s
 	}
 
 	env := map[string]string{
+		"HOME":                 os.Getenv("HOME"),
+		"ASDF_DIR":             os.Getenv("ASDF_DIR"),
```

Comment:
`ASDF_DIR` isn't something that is used in asdf 0.16.0+. Are there still plugins relying on this?

Reply:
> Are there still plugins relying on this?

Yes : https://github.com/asdf-vm/asdf-nodejs/blob/93bd217ba74f05080eebb7d8e1c146a385b01d7f/lib/utils.sh#L24

Reply:
Is there a reason why we shouldn't propagate all env variables of the current process by default here (similar to https://github.com/asdf-vm/asdf/pull/1958 )?

Reply:
ping @Stratus3D thank you!